### PR TITLE
Batch processing event and people update

### PR DESCRIPTION
**Pull request by rakvat**
I build on https://github.com/craig-miskell-fluxfederation/fluent-plugin-mixpanel/tree/batching_mixpanel_calls and added the possibility to send profile updates.

### CODE REVIEW

**Comment by rakvat** [comment]
[[Analytics] Make MixPanel ASYNC](https://trello.com/c/paELJKue/985-analytics-make-mixpanel-async)

**Review by ErebusBat — APPROVED** [review]
Why the change to single quotes?

File: lib/fluent/plugin/out_mixpanel.rb [hunk 1]
```diff
@@ -44,7 +48,14 @@ def configure(conf)
   def start
     super
     error_handler = Fluent::MixpanelOutputErrorHandler.new(log)
-    @tracker = Mixpanel::Tracker.new(@project_token, error_handler)
+    if(@batch_to_mixpanel)
```
Comment (reneklacan) [inline lib/fluent/plugin/out_mixpanel.rb]: this is not javascript :D

File: lib/fluent/plugin/out_mixpanel.rb [hunk 2]
```diff
@@ -44,7 +48,14 @@ def configure(conf)
...
+      @batched_consumer = Mixpanel::BufferedConsumer.new
+      @tracker = Mixpanel::Tracker.new(@project_token, error_handler) do | type, message |
```
Comment (reneklacan) [inline lib/fluent/plugin/out_mixpanel.rb]: weird spacing between `|`

File: lib/fluent/plugin/out_mixpanel.rb (hunk 1 above)
Comment (reneklacan) [reply lib/fluent/plugin/out_mixpanel.rb]: ah this is not your code, don't worry then

Comment (reneklacan) [inline lib/fluent/plugin/out_mixpanel.rb]: `if()`?

**Review by reneklacan — APPROVED** [review]

**Pull request merged**